blueprints/cloud-operations/network-dashboard/README.md

```diff
@@ -46,22 +46,25 @@ The Cloud Function currently tracks usage, limit and utilization of:
 - internal forwarding rules for internal L7 load balancers per VPC
 - internal forwarding rules for internal L4 load balancers per VPC peering group
 - internal forwarding rules for internal L7 load balancers per VPC peering group
-- Dynamic routes per VPC
-- Dynamic routes per VPC peering group
+- Dynamic routes per VPC (note: assumes global routing is ON)
+- Dynamic routes per VPC peering group (note: assumes custom routes importing/exporting is ON)
+- Static routes per project (VPC drill down is available for usage)
+- Static routes per VPC peering group (note: assumes custom routes sharing is ON for all peered networks)
 - IP utilization per subnet (% of IP addresses used in a subnet)
 - VPC firewall rules per project (VPC drill down is available for usage)
 - Tuples per Firewall Policy
 
 It writes this values to custom metrics in Cloud Monitoring and creates a dashboard to visualize the current utilization of these metrics in Cloud Monitoring.
 
-Note that metrics are created in the cloud-function/metrics.yaml file.
+Note that metrics are created in the cloud-function/metrics.yaml file. also note that the Cloud Function assumes all VPCs in peering groups are within the same organization. 
```

Comment:
Why this remark about the same organization?

Reply:
On paper, it is possible to peer VPCs from other orgs, now that I think of it, it should work fine if permissions are right. how about we peer a VPC between our test orgs and see it working e2e? then once we're sure we can change this to something like "it is assumed the CF service account is granted permissions in external orgs, those permissions are not set by the TF in this repo"